pyproject.toml

```diff
@@ -5,13 +5,14 @@ description = ""
 authors = ["Sasha Weinstein <70385853+SashaWeinstein@users.noreply.github.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9,<3.9.7 || >3.9.7,<4.0"
 pandas = "^1.4.3"
 numpy = "^1.23.1"
-streamlit = "plotly"
```

Comment:
Were you able to get to the bottom of why  this happened?

Reply:
lol no. I didn't look that hard though, could comb through commit history more carefully